services/user/Events/include/services/user/REvents.hpp

```diff
@@ -11,9 +11,13 @@ namespace UserService
    struct REvents : psibase::Service<REvents>
    {
       static constexpr psibase::AccountNumber service{"r-events"};
+
+      // Synchronous calls between other services
+      std::vector<char> sqlQuery(const std::vector<char>& query);
```

Comment:
`std::string` is probably more convenient for the caller.

Reply:
done